pkg/operator/reports.go

```diff
@@ -734,6 +742,73 @@ func (op *Reporting) runReport(logger log.FieldLogger, report *metering.Report)
 	return nil
 }
 
+func deleteReport(report *metering.Report, op *Reporting) {
```

Comment:
If you're passing `op* Reporting`, I would investigate making this a method of that type instead of a function that takes that type as an argument.

Reply:
yep moving to using op.deleteReport and removing the one that was added